### ISSUE TITLE
Remove Softmax from CEILing policy

### ISSUE THREAD

**Issue by rafaelBauer** [rafaelBauer/neuro-ceiling#70]
We are using the `CrossEntropyLoss` from Pytorch, therefore the Softmax is not needed in the policy.

From the documentation of [`CrossEntropyLoss`](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html#torch.nn.CrossEntropyLoss) :
"Note that this case is equivalent to applying [LogSoftmax](https://pytorch.org/docs/stable/generated/torch.nn.LogSoftmax.html#torch.nn.LogSoftmax) on an input, followed by [NLLLoss](https://pytorch.org/docs/stable/generated/torch.nn.NLLLoss.html#torch.nn.NLLLoss)."
